### DISCUSSION TITLE
Can't claim student azure pack

### DISCUSSION THREAD

**Discussion by KaymMikael** [community/community#141529] [GitHub Education]
I want to claim the Student Azure Pack so i register my github account to education github then github approves my request. when I was claiming the pack I signed in using my github account then it redirect me to the azure portal but then it still says I dont have any subscription, at education github dashboard it says that I already claimed the azure pack

**Comment by Uchemena** [comment]
Hi, write to Azure support on twitter.

**Comment by davevad93** [comment]
Hi @KaymMikael ,

This is a known issue, #131336, #135739, #140395, #140678, to solve it you can do the following:

1. Contact GitHub support by opening a ticket [here](https://support.github.com/contact/education), select `I am a verified student having trouble redeeming my Global Campus offers` and explain your problem with Azure verification. 

Or

2.  Create a support ticket directly on [Azure](https://azure.microsoft.com/en-us/support/create-ticket).

I personally recommend the first option.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @KaymMikael ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
